Failed to publish to docker hub

**Describe the bug**

As in https://github.com/ChainSafe/lodestar/runs/3166394582

```
Step 10/11 : RUN /usr/app/node_modules/.bin/lodestar --help
 ---> Running in b9bfac1d8b3a
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '../../../utils/lib'
```

**Expected behavior**

Should be able to publish to docker hub

--- a/packages/lodestar/src/chain/opPools/aggregatedAttestationPool.ts
+++ b/packages/lodestar/src/chain/opPools/aggregatedAttestationPool.ts
@@ -1,6 +1,7 @@
 import bls from "@chainsafe/bls";
 import {ForkName, MAX_ATTESTATIONS, MIN_ATTESTATION_INCLUSION_DELAY, SLOTS_PER_EPOCH} from "@chainsafe/lodestar-params";
-import {allForks, altair, Epoch, Slot, ssz, ValidatorIndex} from "@chainsafe/lodestar-types";
+import {altair, Epoch, Slot, ssz, ValidatorIndex} from "@chainsafe/lodestar-types";
+import {allForks} from "@chainsafe/lodestar-beacon-state-transition";
 import {
   CachedBeaconState,
   computeEpochAtSlot,
@@ -11,7 +12,6 @@ import {List, readonlyValues, toHexString} from "@chainsafe/ssz";
 import {MapDef} from "../../util/map";
 import {pruneBySlot} from "./utils";
 import {InsertOutcome} from "./types";
-import {EpochContext} from "../../../../beacon-state-transition/lib/allForks";
 
 type DataRootHex = string;
 
@@ -331,7 +331,7 @@ export function aggregateInto(
 
 export function extractParticipation(
   attestations: List<phase0.PendingAttestation>,
-  epochCtx: EpochContext
+  epochCtx: allForks.EpochContext
 ): Set<ValidatorIndex> {
   const allParticipants = new Set<ValidatorIndex>();
   for (const att of readonlyValues(attestations)) {

--- a/packages/lodestar/src/db/interface.ts
+++ b/packages/lodestar/src/db/interface.ts
@@ -3,6 +3,7 @@
  */
 
 import {allForks} from "@chainsafe/lodestar-types";
+import {IDbMetrics} from "@chainsafe/lodestar-db";
 
 import {
   AttesterSlashingRepository,
@@ -26,7 +27,6 @@ import {
   LatestNonFinalizedUpdate,
 } from "./single";
 import {PendingBlockRepository} from "./repositories/pendingBlock";
-import {IDbMetrics} from "../../../db/lib";
 
 /**
  * The DB service manages the data layer of the beacon chain

--- a/packages/lodestar/src/eth1/provider/eth1Provider.ts
+++ b/packages/lodestar/src/eth1/provider/eth1Provider.ts
@@ -1,6 +1,7 @@
 import {fromHexString, toHexString} from "@chainsafe/ssz";
 import {phase0} from "@chainsafe/lodestar-types";
 import {AbortSignal} from "@chainsafe/abort-controller";
+import {IChainForkConfig} from "@chainsafe/lodestar-config";
 import {chunkifyInclusiveRange} from "../../util/chunkify";
 import {linspace} from "../../util/numpy";
 import {retry} from "../../util/retry";
@@ -8,7 +9,6 @@ import {ErrorParseJson, JsonRpcHttpClient} from "./jsonRpcHttpClient";
 import {depositEventTopics, parseDepositLog} from "../utils/depositContract";
 import {IEth1Provider} from "../interface";
 import {IEth1Options} from "../options";
-import {IChainForkConfig} from "../../../../config/lib";
 import {isValidAddress} from "../../util/address";
 
 /* eslint-disable @typescript-eslint/naming-convention */

--- a/packages/lodestar/src/metrics/metrics.ts
+++ b/packages/lodestar/src/metrics/metrics.ts
@@ -6,7 +6,7 @@ import {IChainForkConfig} from "@chainsafe/lodestar-config";
 import {allForks} from "@chainsafe/lodestar-types";
 import {collectDefaultMetrics, Counter, Registry} from "prom-client";
 import gcStats from "prometheus-gc-stats";
-import {DbMetricLabels, IDbMetrics} from "../../../db/lib";
+import {DbMetricLabels, IDbMetrics} from "@chainsafe/lodestar-db";
 import {createBeaconMetrics, IBeaconMetrics} from "./metrics/beacon";
 import {createLodestarMetrics, ILodestarMetrics} from "./metrics/lodestar";
 import {IMetricsOptions} from "./options";

--- a/packages/lodestar/src/network/interface.ts
+++ b/packages/lodestar/src/network/interface.ts
@@ -2,7 +2,7 @@
  * @module network
  */
 import {Connection} from "libp2p";
-import {ENR} from "@chainsafe/discv5/lib";
+import {ENR} from "@chainsafe/discv5";
 import Multiaddr from "multiaddr";
 import PeerId from "peer-id";
 import {INetworkEventBus} from "./events";

--- a/packages/lodestar/src/util/retry.ts
+++ b/packages/lodestar/src/util/retry.ts
@@ -1,4 +1,4 @@
-import {sleep} from "../../../utils/lib";
+import {sleep} from "@chainsafe/lodestar-utils";
 
 export interface IRetryOptions {
   /**

